static/src/javascripts/projects/common/utils/$.js

```diff
@@ -0,0 +1,33 @@
+import bonzo from 'bonzo';
+import qwery from 'qwery';
+
+// Warning: side effect. This patches the bonzo module for use everywhere
+bonzo.aug({
+    height() {
+        return this.dim().height;
+    },
+});
+
+function $(selector, context) {
+    return bonzo(qwery(selector, context));
+}
+
+$.create = s => bonzo(bonzo.create(s));
+
+$.ancestor = (el, c) => {
+    if (el.nodeName.toLowerCase() === 'html') {
+        return false;
+    }
+    if (!el.parentNode || bonzo(el.parentNode).hasClass(c)) {
+        return el.parentNode;
+    }
+    return $.ancestor(el.parentNode, c);
+};
+
+$.forEachElement = (selector, fn) => {
+    const els = qwery(selector);
+    els.forEach(fn);
+    return els;
+};
```

Comment:
I just checked, how often `forEachElement` is used. Turns out: 7 times, so it's not as critical as I thought.

1. What do you think about taking it out of `utils/$` and move it to it's own utility, to clearify, it's not part of `bonzo`?

2. I'm not very opinionated about whether to use `map` for `forEach` in this case. I'm asking myself, why it exists anywy, since there is [bonzo().each()](https://github.com/ded/bonzo#api-each).

3. 🐎 Let's not fall into a micro optimization trap for now - just make `forEachElement` work - my fault.